### ISSUE TITLE
Uninitialized constant PaperTrail::VersionAssociation

### ISSUE THREAD

**Issue by Abrissirba** [airblade/paper_trail#537]
I get the following error when I upgrade from 4.0.0.beta to 4.0.0.rc1. 

```ruby
c:/Ruby200-x64/lib/ruby/gems/2.0.0/gems/paper_trail-4.0.0.rc1/lib/paper_trail/config.rb:33:in `track_associations': uninitialized constant PaperTrail::VersionAssociation (NameError)
        from c:/Ruby200-x64/lib/ruby/gems/2.0.0/gems/paper_trail-4.0.0.rc1/lib/paper_trail/version_concern.rb:14:in `block in <module:VersionConcern>'
        from c:/Ruby200-x64/lib/ruby/gems/2.0.0/gems/activesupport-4.2.1/lib/active_support/concern.rb:120:in `class_eval'
        from c:/Ruby200-x64/lib/ruby/gems/2.0.0/gems/activesupport-4.2.1/lib/active_support/concern.rb:120:in `append_features'
        from c:/Ruby200-x64/lib/ruby/gems/2.0.0/gems/paper_trail-4.0.0.rc1/lib/paper_trail/frameworks/active_record/models/paper_trail/version.rb:5:in `include'
        from c:/Ruby200-x64/lib/ruby/gems/2.0.0/gems/paper_trail-4.0.0.rc1/lib/paper_trail/frameworks/active_record/models/paper_trail/version.rb:5:in `<class:Version>'
        from c:/Ruby200-x64/lib/ruby/gems/2.0.0/gems/paper_trail-4.0.0.rc1/lib/paper_trail/frameworks/active_record/models/paper_trail/version.rb:4:in `<module:PaperTrail>'
        from c:/Ruby200-x64/lib/ruby/gems/2.0.0/gems/paper_trail-4.0.0.rc1/lib/paper_trail/frameworks/active_record/models/paper_trail/version.rb:3:in `<top (required)>'
        from c:/Ruby200-x64/lib/ruby/gems/2.0.0/gems/paper_trail-4.0.0.rc1/lib/paper_trail/frameworks/active_record.rb:4:in `require'
        from c:/Ruby200-x64/lib/ruby/gems/2.0.0/gems/paper_trail-4.0.0.rc1/lib/paper_trail/frameworks/active_record.rb:4:in `block in <top (required)>'
        from c:/Ruby200-x64/lib/ruby/gems/2.0.0/gems/paper_trail-4.0.0.rc1/lib/paper_trail/frameworks/active_record.rb:3:in `each'
        from c:/Ruby200-x64/lib/ruby/gems/2.0.0/gems/paper_trail-4.0.0.rc1/lib/paper_trail/frameworks/active_record.rb:3:in `<top (required)>'
        from c:/Ruby200-x64/lib/ruby/gems/2.0.0/gems/paper_trail-4.0.0.rc1/lib/paper_trail.rb:155:in `require'
        from c:/Ruby200-x64/lib/ruby/gems/2.0.0/gems/paper_trail-4.0.0.rc1/lib/paper_trail.rb:155:in `<top (required)>'
        from c:/Ruby200-x64/lib/ruby/gems/2.0.0/gems/bundler-1.7.9/lib/bundler/runtime.rb:76:in `require'
        from c:/Ruby200-x64/lib/ruby/gems/2.0.0/gems/bundler-1.7.9/lib/bundler/runtime.rb:76:in `block (2 levels) in require'
        from c:/Ruby200-x64/lib/ruby/gems/2.0.0/gems/bundler-1.7.9/lib/bundler/runtime.rb:72:in `each'
        from c:/Ruby200-x64/lib/ruby/gems/2.0.0/gems/bundler-1.7.9/lib/bundler/runtime.rb:72:in `block in require'
        from c:/Ruby200-x64/lib/ruby/gems/2.0.0/gems/bundler-1.7.9/lib/bundler/runtime.rb:61:in `each'
        from c:/Ruby200-x64/lib/ruby/gems/2.0.0/gems/bundler-1.7.9/lib/bundler/runtime.rb:61:in `require'
        from c:/Ruby200-x64/lib/ruby/gems/2.0.0/gems/bundler-1.7.9/lib/bundler.rb:134:in `require'
        from C:/Projects/Comic Health/api/config/environment.rb:5:in `<top (required)>'
        from c:/Ruby200-x64/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require'
        from c:/Ruby200-x64/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require'
        from C:/Projects/Comic Health/api/test/test_helper.rb:3:in `<top (required)>'
        from c:/Ruby200-x64/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require'
        from c:/Ruby200-x64/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require'
        from C:/Projects/Comic Health/api/test/apis/cards_spec.rb:1:in `<top (required)>'
        from c:/Ruby200-x64/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require'
        from c:/Ruby200-x64/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require'
        from c:/Ruby200-x64/lib/ruby/gems/2.0.0/gems/rake-10.4.2/lib/rake/rake_test_loader.rb:15:in `block in <main>'
        from c:/Ruby200-x64/lib/ruby/gems/2.0.0/gems/rake-10.4.2/lib/rake/rake_test_loader.rb:4:in `select'
        from c:/Ruby200-x64/lib/ruby/gems/2.0.0/gems/rake-10.4.2/lib/rake/rake_test_loader.rb:4:in `<main>'
```

**Comment by Kiranchang** [comment]
I am getting the same error message when I upgrade from 4.0.0.beta to 4.0.0.rc.

**Comment by batter** [comment]
When you ran the install generator, did you run it with the `--with-associations` flag?

`rails generate paper_trail:install --with-associations`

**Comment by ipcloud** [comment]
I'm using paper_trail in a grape application without rails so I'm not using rails generate.

**Comment by Abrissirba** [comment]
I'm using paper_trail in a grape application without rails so I'm not using rails generate

**Comment by batter** [comment]
You need to create a [`versions_associations` table](https://github.com/airblade/paper_trail/blob/master/lib/generators/paper_trail/templates/create_version_associations.rb) if you haven't already?

**Comment by batter** [comment]
This should be addressed / fixed by the merge of #554. Please try bundling from source and let me know if that doesn't fix your issue.
